Automattic/newspack-ads
# feat(media-kit): add media kit (from publisher-media-kit plugin)

### All Submissions:






### Changes proposed in this Pull Request:



Incorporates a Media Kit page functionality, adapted from the https://github.com/10up/publisher-media-kit/ plugin:

![media-kit](https://github.com/user-attachments/assets/7a9fbded-8f81-4903-9cd7-3814aa2820b9)

### How to test the changes in this Pull Request:

1. If `publisher-media-kit` plugin is active, deactivate it and ensure the media kit page and `pmk-page` option are deleted
2. Use the WP CLI command `wp newspack-ads create-media-kit-page` to create the page, observe the command outputs the created post ID
3. Run the command again, observe it informs that the page has already been created, and outputs the post ID
4. Observe a media kit page draft was created, edit it 
5. Observe it has the content as in the image seen above
6. Publish the page, observe the interactive elements – the tabs in "Ad Specs" and "Our Rates" sections – are functioning properly

### Other information:




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207910087478332
